config/data.js

```diff
@@ -1762,6 +1762,7 @@ module.exports = {
       y_2016: 2.5,
       y_2017: 2.3,
       y_2018: 2.3,
+      y_2019: 2.4,
```

Comment:
@Vanie2020 where is the BUSWK world value?

Reply:
Ill push it again. should be in this branch

Reply:
great, thx!